registration-processor/core-processor/registration-processor-abis-handler-stage/src/main/java/io/mosip/registration/processor/abis/handler/stage/AbisHandlerStage.java

```diff
@@ -593,20 +587,15 @@ private String getDataShareUrl(String id, String process) throws Exception {
 
 		Map<String, String> tags = packetManagerService.getAllTags(id);
 		String ageGroup = tags.get("AGE_GROUP");
-
-		if (ageGroup.equalsIgnoreCase("INFANT")) {
-			validateBiometricRecord(biometricRecord, modalities, biometricModalitySegmentsMapInfant,
-					priorityBasedPacketManagerService.getMetaInfo(id, process, ProviderStageName.BIO_DEDUPE));
-		} else if (ageGroup.equalsIgnoreCase("MINOR")) {
-			validateBiometricRecord(biometricRecord, modalities, biometricModalitySegmentsMapMinor,
-					priorityBasedPacketManagerService.getMetaInfo(id, process, ProviderStageName.BIO_DEDUPE));
-		} else if (ageGroup.equalsIgnoreCase("ADULT")) {
-			validateBiometricRecord(biometricRecord, modalities, biometricModalitySegmentsMapAdult,
-					priorityBasedPacketManagerService.getMetaInfo(id, process, ProviderStageName.BIO_DEDUPE));
-		} else {
-			validateBiometricRecord(biometricRecord, modalities, biometricModalitySegmentsMapAdult,
-					priorityBasedPacketManagerService.getMetaInfo(id, process, ProviderStageName.BIO_DEDUPE));
+		Map<String, List<String>> biometricModalitySegmentsMap;
+		if(biometricModalitySegmentsMapforAgeGroup.containsKey(ageGroup)){
+			biometricModalitySegmentsMap = biometricModalitySegmentsMapforAgeGroup.get(ageGroup);
+			}
+		else {
+			biometricModalitySegmentsMap = biometricModalitySegmentsMapforAgeGroup.get("DEFAULT");
```

Comment:
if the map does not contain default age group, how are we handling that ?

Reply:
For Else condition we assumed always DEFAULT should be present  and proceeded without adding any null check.

Reply:
As discussed, @sowmya695 to create a Jira ticket to handle this case.